library/blivet.py

```diff
@@ -755,12 +763,11 @@ def _create_members(self):
 
 
         if self._is_raid:
-            raid_name = "%s-1" % self._pool['name']
-
-            raid_array = self._new_mdarray(members, raid_name=raid_name)
+            raid_array = self._new_mdarray(members, raid_name=self._pool["raid_name"])
 
             self._blivet.create_device(raid_array)
             result = [raid_array]
+            self._pool['raid_name'] = raid_array.name
```

Comment:
is it expected that you first use `self._pool["raid_name"]` and then you set it?